README.rst

```diff
@@ -204,6 +204,13 @@ emails are affected and not the local git commits::
 
   $ git publish --to patches@example.org --signoff
 
+git-publish supports the --dry-run option which is passed through to
+git-send-email to see what would be sent without actually sending anything::
```

Comment:
> This means the CC list may contain sendemail.aliasesFile aliases instead of real email addresses

More importantly, it's missing the email addresses added automatically by git-send-email due to the various Signed-off-by's. Are they supposed to be added by options.cc_cmd somehow?

> Did you consider modifying inspect_menu() to call git-send-email --dry-run so the user can see the final email output? I think that would make the existing --inspect-emails option more useful.

I did not, --dry-run was my first choice coming from git-send-email. I will extend inspect_menu() if you prefer that. Thank you!